### ISSUE TITLE
Accessibility Inputs do not have autocomplete attribute

### ISSUE THREAD

**Issue by msiewierska** [shopware/platform#14017]
### Shopware Version

6.7.3.1

### Affected area / extension

Platform(Default)

### Actual behaviour

Some fields in different forms have no autocomplete attribute:

![Image](https://github.com/user-attachments/assets/db5edd02-1bca-45f9-abb1-fb362e6430f2)

<img width="1440" height="177" alt="Image" src="https://github.com/user-attachments/assets/8aec7bb3-a0fa-49c8-8bd6-c6e06a57160c" />

Missing autocomplete breaks criteria 1.3.5 (level AA)

### Expected behaviour

For users who use assistive technologies, a valid autocomplete value helps the technology to understand the form fields correctly. Without valid values, users may become confused, experience delays, or make mistakes when they fill out forms.

All inputs should have autocomplete attribute.

If there is proper autocomplete value for a give input (e.g. coupon code), there should be autocomplete="off" attribute present

### How to reproduce

Inspect form and inputs across the shop.
